src/core/utils/components/Popover.tsx

```diff
@@ -56,6 +57,15 @@ export const Popover = React.forwardRef((props: PopoverProps, ref) => {
     }),
   };
 
+  // Plugin to revert default tippy styles by applying iui-popover class to tippy-box
+  const revertTippyStyles = {
```

Comment:
You can just add `className` to `<Tippy />`

Reply:
Done